roles/nuage_master/defaults/main.yml

```diff
@@ -2,7 +2,13 @@
 r_nuage_master_firewall_enabled: "{{ os_firewall_enabled | default(True) }}"
 r_nuage_master_use_firewalld: "{{ os_firewall_use_firewalld | default(False) }}"
 
-nuage_mon_rest_server_port: '9443'
+nuage_mon_rest_server_port: "{{ nuage_openshift_monitor_rest_server_port | default('9443') }}"
+
+nuage_mon_image_name: "nuage/master"
```

Comment:
It should hardcode the registry here (`docker.io/` I assume), so that RHEL machines won't look these up at `registry.access.redhat.io` and other repos.

Reply:
They are neither pulled from `docker.io` nor from `registry.access.redhat.io`. They are expected to be loaded on the machine by the user. But, anyways I modifed the variable so that the user has the ability to pass the registry name as well.